### PR TITLE
Better document Actions and Events in AMP

### LINKED ISSUE #5641: Consider exposing an explicit `open` action on lightbox
Having `close` action but relying on the default `activate` trigger for an `open` feels weird sometimes. The expectation is if I can do `close` I should be able to do `open`.

This is pretty simple to do. Just need to add an alias `open` for activate method.

### PR DESCRIPTION
**Pull request by mkhatib**
Closes #5641 and #5617

### CODE REVIEW

File: extensions/amp-image-lightbox/amp-image-lightbox.md [hunk 1]
```diff
@@ -75,6 +75,20 @@ properties that can be styled are `background` and `color`.
...
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>(default)</td>
+    <td>Opens the image lightbox with the source image being the one that triggered the action.</td>
+  </tr>
+</table>
+
```
Comment (camelburrito) [inline extensions/amp-image-lightbox/amp-image-lightbox.md]: i think it would be useful to provide how to use examples here.

Comment (camelburrito) [inline extensions/amp-image-lightbox/amp-image-lightbox.md]: (same below)

File: spec/amp-html-format.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ HTML tags can be used unchanged in AMP HTML. Certain tags have equivalent custom
   </tr>
   <tr>
     <td width="30%"><code><a name="ancr"></a>a</code></td>
-    <td>The <code>href</code> attribute value must not begin with <code>javascript:</code>. If set, the <code>target</code> attribute value must be <code>_blank</code>. Otherwise allowed. <a href="#ancr">🔗</a></td>
+    <td>The <code>href</code> attribute value must not begin with <code>javascript:</code>. If set, the <code>target</code> attribute value must be <code>_blank</code> or `_top`. Otherwise allowed. <a href="#ancr">🔗</a></td>
```
Comment (cramforce) [inline spec/amp-html-format.md]: I don't think it is rolled out yet, but @Gregable would know.

**Comment by mkhatib** [comment]
@camelburrito 👀 PTAL

File: spec/amp-html-format.md [hunk 3]
```diff
@@ -168,11 +168,11 @@ HTML tags can be used unchanged in AMP HTML. Certain tags have equivalent custom
   </tr>
   <tr>
     <td width="30%">form</td>
-    <td>Prohibited. <a href="https://github.com/ampproject/amphtml/issues/1286">Support coming in the future</a>.</td>
+    <td>Allowed. Require including [amp-form extension](../extensions/amp-form/amp-form.md).</td>
   </tr>
   <tr>
     <td width="30%">input elements</td>
-    <td>Prohibited. Includes <code>input</code>, <code>textarea</code>, <code>select</code>, <code>option</code>. Notably, the <code>button</code> element is allowed.</td>
+    <td>Mostly Allowed with exception of some input types, namely, `input[type=image]`, `input[type=button]`, `input[type=password]`, `input[type=file]` are invalid. Related tags are also allowed: `fieldset`, `label`</td>
```
Comment (camelburrito) [inline spec/amp-html-format.md]: exception of some input types - link it to the right section in form.md

**Pull request merged**